docs/MO_DG/prepare_model/convert_model/Convert_Model_From_TensorFlow.md

```diff
@@ -336,7 +336,7 @@ TensorFlow*-specific parameters:
 
 ## Convert TensorFlow* 2 Models <a name="Convert_From_TF2X"></a>
 
-In order to convert TensorFlow* 2 models, installation of dependencies from `requirements_tf2.txt` is required.
+In order to convert TensorFlow* 2 models, installation of dependencies from `requirements_tf.txt` is required.
```

Comment:
@slyubimt is the file name aligned with AC?

Reply:
There is no policy for naming requirements files right now. requirements-tensorflow.in used for `tensorflow2` extra tag for OMZ tools: https://github.com/openvinotoolkit/open_model_zoo/blob/ee423afb0d63d980b1d353dd240c5658483244cc/tools/model_tools/setup.py#L101

@jkamelin , @eaidova Do you have any preferences for this?

Reply:
Now we have both requirements_tf2.txt and requirements_tf.txt with same requirements